### ISSUE TITLE
`webhook_url` is not mandatory in garm `v0.1.4` but needed by updating to `v0.1.5`

### ISSUE THREAD

**Issue by bavarianbidi** [cloudbase/garm#311]
in garm `v0.1.4` it's not mandatory to have `webhook_url` set.

By updating garm from `v0.1.4` to `v0.1.5`, the `/controller-info` endpoint will fail as `webhook_url` got not converted into the database.

```bash
garm-cli-v0.1.5 controller show
Error: [GET /controller-info][409] controllerInfoConflict {"error":"urls_required","details":"Missing required URLs. Make sure you update the metadata, callback and webhook URLs"}
```

**Comment by gabriel-samfira** [comment]
Right. In it's current form (no scale sets yet), the webhook_url is kind of mandatory. We should probably relax this, as it won't really be mandatory once we have scale sets. It will only be mandatory for the current pools. So we can probably set this to optional.
